pkg/sql/colexec/cfetcher.go

```diff
@@ -69,6 +69,9 @@ type cTableInfo struct {
 	// One value per column that is part of the key; each value is a column
 	// index (into cols); -1 if we don't need the value for that column.
 	indexColOrdinals []int
+	// allIndexColOrdinals is the same as allIndexColOrdinals but
+	// does not contain any -1's. It is meant to be used only in logging.
+	allIndexColOrdinals []int
```

Comment:
It seems like we use it for more than just logging, no?

Reply:
we use it to just fill in all of the index columns when printing k/vs to the trace, as opposed to the needed set when tracekv is off